portfolio.html

```diff
@@ -6,21 +6,25 @@
 		<title>My Portfolio</title>
 		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
 	</head>	
+	
 	<body>
+		
 		<nav class="nav_bar">
 			<div><label class="label_karen_schulz" for="nav_bar">Karen Schulz</label></div>
 			<div><a href="#">Home</a>
 			<a href="#">Contact</a></div>
 		</nav>
-		<div class="welcome"><img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture">
-			<section>
-			<div class="heading"><h2>Welcome to my Portfolio</h2></div>
-			<div class="portfolio1"><img class="first_work" src="images/green_back.png" alt="my_first_work"><p>I love to do illustrations, check my creations</p>
```

Comment:
These div don't seem to have a </div>, also the </section> tag needs indenting

Reply:
I went over indentings and tidied them up. I also tidied that bit of coding.